templates/webapi/comments/comment_row.html.ep

```diff
@@ -26,7 +26,7 @@
                         % if($comment) {
                             <%= $user->name %> wrote
                             <a href="#comment-<%= $comment_id %>" class="comment-anchor"><abbr class="timeago" title="<%= $comment->t_created->datetime() %>Z"><%= format_time($comment->t_created) %></abbr></a>
-                            % if ($comment->t_created != $comment->t_updated) {
+                            % if (($comment->t_updated - 1000) >= $comment->t_created) {
```

Comment:
`t_updated` is supposed to be a `DateTime` object so you can not just substract 1000. I suppose you need to substract a `DateTime::Duration` object.

Reply:
```perl
my $updated = $comment->t_updated;
$updated->subtract(seconds => DB_TIMESTAMP_ACCURACY);
if ($updated >= $comment->t_created) {
```

You can probably do this? Also using a constant rather than a magic second?

Reply:
yes, done like that, kinda